src/fiobank/fiobank.py

```
@@ -72,11 +73,8 @@ def info_schema(self) -> None:
         stop=stop_after_attempt(3),
         wait=wait_random_exponential(max=2 * 60),
     )
-    def _request(self, action: str, **params) -> dict | None:
-        url_template = self.base_url + self.actions[action]
-        url = url_template.format(token=self.token, **params)
-
-        response = requests.get(url)
+    def _get(self, url: str, params: dict | None = None) -> requests.Response:
+        response = requests.get(url, params=params)
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)